.eslintrc.yml

```diff
@@ -1,4 +1,4 @@
-extends: seegno
+extends: uphold
 env:
   jasmine: true
   jest: true
```

Comment:
This entire file seems [redundant](https://github.com/uphold/eslint-config-uphold/blob/v1.0.1/src/index.js#L6-L12).

Reply:
Nice catch @waldyrious ;)